linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml

```diff
@@ -10,14 +10,19 @@ rationale: |-
     Removing the <tt>vsftpd</tt> package decreases the risk of its
     accidental activation.
 
-severity: low
+severity: {{% if product == "ol7" -%}}
```

Comment:
Another way how to do things like this is a rule refinement in a profile - for example here: https://github.com/ComplianceAsCode/content/blob/master/rhel8/profiles/ospp.profile#L125 I find refinements preferable, as changing the default severity of the rule may have consequences down the road.

Reply:
@matejak thanks for the hint, makes sense to me. Really, I've seen this adjustment as an example and profile looks like more reasonable place for this. I'll update this PR to move severity to the profile.

Note: Making this change I just followed another existing approach:  https://github.com/ComplianceAsCode/content/blob/c4100ac8ba1c255dd8df9bf9cf191e7b850b4eef/linux_os/guide/services/obsolete/tftp/tftpd_uses_secure_mode/rule.yml#L19
Probably it worth to update places like these as well to avoid confusion and note severity adjustment howto in the Developers Guide (if not done already:))?

Reply:
@matejak I've noticed refined severity taken into account in eval scan results/report only. Generated guide uses rule default value. Is it by design, as it sounds reasonable to see refinements in generated guides as well?

Reply:
As guides are profile-specific, I would expect them to show the same severity as the scanner does during the scan. In other words, it looks like to be a bug in the scanner.

Reply:
Opened new issue for tracking: https://github.com/OpenSCAP/openscap/issues/1512